tests/test_project.py

```diff
@@ -39,6 +41,16 @@
     _switch_to_directory,
 )
 
+try:
+    import git
+
+    skip_git = False
+except ImportError:
+    skip_git = True
+
+
+git_mark_skipif = pytest.mark.skipif(skip_git, reason="git could not be imported")
```

Comment:
Pytest also has a feature called importorskip that could be helpful, but I’m fine with the current design here.